op-chain-ops/genesis/config.go

```diff
@@ -790,6 +824,31 @@ func NewL2StorageConfig(config *DeployConfig, block *types.Block) (state.Storage
 		"_name":   "Wrapped Ether",
 		"_symbol": "WETH",
 	}
+	storage["L2UsdcBridge"] = state.StorageValues{
+		"messenger":          predeploys.L2CrossDomainMessengerAddr,
+		"otherBridge":        config.L1UsdcBridgeAddr,
+		"l1Usdc":             config.L1UsdcAddr,
+		"l2Usdc":             predeploys.FiatTokenV2_2Addr,
+		"l2UsdcMasterMinter": predeploys.MasterMinterAddr,
+	}
+	storage["SignatureChecker"] = state.StorageValues{}
+	storage["MasterMinter"] = state.StorageValues{
+		"_owner":        config.MasterMinterOwner,
+		"controllers":   controllers,
```

Comment:
```go
"controllers": map[any]any{
    "_controller": predeploys.L2UsdcBridgeAddr,
    "_worker":     predeploys.L2UsdcBridgeAddr,
}
```
How about fix it like this? 
Since `controllers` is only for the L2UsdcBridge, I think it's okay to do this.

Reply:
Thank you good idea. I will fix it!!